sympy/physics/continuum_mechanics/truss.py

```diff
@@ -198,46 +203,48 @@ def remove_node(self, label):
 
         >>> from sympy.physics.continuum_mechanics.truss import Truss
         >>> t = Truss()
-        >>> t.add_node('A', 0, 0)
+        >>> t.add_node(('A', 0, 0), ('B', 3, 0), ('C', 5, 0))
         >>> t.nodes
-        [('A', 0, 0)]
-        >>> t.add_node('B', 3, 0)
-        >>> t.nodes
-        [('A', 0, 0), ('B', 3, 0)]
-        >>> t.remove_node('A')
+        [('A', 0, 0), ('B', 3, 0), ('C', 5, 0)]
+        >>> t.remove_node('A', 'C')
         >>> t.nodes
         [('B', 3, 0)]
         """
-        for i in range(len(self.nodes)):
-            if self._node_labels[i] == label:
-                x = self._node_position_x[i]
-                y = self._node_position_y[i]
+        for label in args:
+            for i in range(len(self.nodes)):
+                if self._node_labels[i] == label:
+                    x = self._node_position_x[i]
+                    y = self._node_position_y[i]
 
-        if label not in self._node_labels:
-            raise ValueError("No such node exists in the truss")
+            if label not in self._node_labels:
+                raise ValueError("No such node exists in the truss")
 
-        else:
-            members_duplicate = self._members.copy()
-            for member in members_duplicate:
-                if label == self._members[member][0] or label == self._members[member][1]:
-                    raise ValueError("The node given has members already attached to it")
-            self._nodes.remove((label, x, y))
-            self._node_labels.remove(label)
-            self._node_positions.remove((x, y))
-            self._node_position_x.remove(x)
-            self._node_position_y.remove(y)
-            if label in list(self._loads):
-                self._loads.pop(label)
-            if label in list(self._supports):
-                self._supports.pop(label)
-            self._node_coordinates.pop(label)
-
-    def add_member(self, label, start, end):
+            else:
+                members_duplicate = self._members.copy()
+                for member in members_duplicate:
+                    if label == self._members[member][0] or label == self._members[member][1]:
+                        raise ValueError("The node given has members already attached to it")
```

Comment:
```suggestion
                        raise ValueError("The given node already has members attached to it")
```
seems more clear and straightforward.

Reply:
Thank you for pointing it out, I have made the change, as suggested